test/integration/multiple_buckets/multiple_buckets_test.go

```diff
@@ -51,6 +51,9 @@ func TestMultipleBuckets(t *testing.T) {
 			case "one":
 				// bucket with suffix one
 				assert.True(op.Get("metadata.iamConfiguration.bucketPolicyOnly.enabled").Bool(), "bucketPolicyOnly is enabled")
+				bucket_lifecycle := op.Get("metadata.lifecycle.rule").Array()[1]
+				assert.Equal("Delete", bucket_lifecycle.Get("action.type").String(), "bucket lifecycle action is Delete")
```

Comment:
Can we add an assert that the other bucket also has the expected rule?

Reply:
No we cannot, because the other bucket (`"two"`) should not have it.
I can try to add an assert that the other bucket doesn't have this rule, but it may be a little bit difficult: should I only assert that `metadata.lifecycle.rule[1]` does not exist, or should I check each element of `metadata.lifecycle.rule` to make sure that it doesn't match the new rule ?

Reply:
@AlexisBRENON sorry I should have clarified - I meant we should check that it only has the one expected SetStorageClass rule. I think asserting length of one for that slice is fine.